src/command.rs

```diff
@@ -149,7 +149,7 @@ pub async fn publish(
     #[cfg(feature = "build")] allow_dirty: bool,
```

Comment:
This has to be the git feature aswell then!

Reply:
Indeed, changed!